src/s1_orbits/s1_orbits.py

```diff
@@ -0,0 +1,41 @@
+import logging
+from pathlib import Path
+from typing import Union
+
+import requests
+
+from .exceptions import InvalidSceneError, OrbitNotFoundError
+
+
+API_URL = "https://s1-orbits.asf.alaska.edu/scene"
+
+
+def fetch_for_scene(
+    scene: str,
+    dir: Union[Path, str] = ".",
+) -> Path:
+    """For the given scene, downloads the AUX_POEORB file if available, otherwise downloads the AUX_RESORB file.
+
+    Args:
+        scene: The scene name for which to download the orbit file.
+        dir: The directory that the orbit file should download into.
+
+    Returns:
+        download_path: The path to the downloaded file.
+    """
+    request_url = f"{API_URL}/{scene}"
+
+    logging.info(f"Downloading {request_url}")
+
```

Comment:
This doesn't print anything for me, I assume because we haven't configured the logger anywhere. I'd be fine with just removing it.
```suggestion
```